src/css/styles.css

```diff
@@ -34,17 +57,23 @@ main {
   flex-direction: column;
   width: 100%;
   height: 100%;
+  background-color: #f2f2f2;
 }
 
 .trip-filter-button-container {
   display: flex;
   justify-content: space-around;
   align-items: center;
   width: 100%;
-  height: 7%;
+  height: 9%;
```

Comment:
I suspect that the percentage height on your button container is what's causing the "squish" as you navigate between filtered views. There might be some default margin or something on an element that's inside the view area.

Reply:
@maddielaw Oops, I just noticed you said the CARD is smaller, not the button bar. It looks like the space between your image and that first line of text gets smaller when there's only one card. How strange - I'm honestly not sure what could be causing that. It looks like some other margins are shifting on the page between views as well.